Lesson26/app/feyza/script.js

```diff
@@ -0,0 +1,38 @@
+// Register on Unsplash to get your own API key
+const API_KEY = 'tmNxD4LLLqpyI4nofipBwzD-Z79nZ-IRKOW2W6sQwCs';
+
+// TODO: Add a new button in HTML to clear the screen
+
+
+const clearButton = document.getElementById("clear-button");
+clearButton.addEventListener("click", function(){
```

Comment:
For clearing the screen, you can select `image-container` div element and set it's textContent to empty string:
`document.getElementById('image-container').textContent = ''`;

Reply:
I'm not sure if this is the case.